pepper-apis/src/main/java/org/broadinstitute/ddp/util/DateTimeUtils.java

```diff
@@ -39,4 +40,8 @@ public static ZoneId parseTimeZone(String givenTimeZone) {
             return null;
         }
     }
+
+    public static Long localDateToEpochSeconds(LocalDate date) {
+        return date == null ? null : date.atStartOfDay(ZoneId.of("UTC")).toEpochSecond();
```

Comment:
Is assuming `UTC` instead of the system's timezone here accurate?

Reply:
Yes, accurate.